style.css

```diff
@@ -1,99 +1,156 @@
 * {
-    margin: 0;
-    padding: 0;
-    box-sizing: content-box;
-    font-family: sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: content-box;
+  font-family: sans-serif;
+}
+
+nav {
+  display: flex;
+  justify-content: center;
+  padding-top: 5px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+nav ul li {
+  list-style-type: none;
+  display: inline-block;
+  padding: 10px 20px;
+}
+
+nav ul li a {
+  color: black;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+.icons a {
+  margin: 0 10px;
+  color: #505f79;
+}
+
+.hero-srolldown a {
+  color: #505f79;
+}
+
+nav ul li a:hover {
+  color: #36b37e;
+  transition: 0.3s;
+}
+
+.nav-list {
+  text-align: center;
+  margin-top: 20px;
+  display: none;
+}
+
+.hero-comtain {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 170px 50px 0 50px;
+  background-image: url("header-illsutration-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 70vh;
+}
```

Comment:
Great effort and improvements 👍🏼  , still have small issues to fixe to be ready for the approval, the top padding of this container is making a huge space on top of the page and also affecting the background alignment kindly fixe it to make your design look like the template as requested by the previous reviewer 

![image (9)](https://github.com/RolandAmini/portfolio-mobile/assets/59707859/026c0c5d-cd19-4666-b2ae-a74d0f08164e)

![image (11)](https://github.com/RolandAmini/portfolio-mobile/assets/59707859/d8b297f9-6e43-4714-9afd-b932d036004d)